src/rtextures.c

```diff
@@ -1494,6 +1498,147 @@ void ImageAlphaPremultiply(Image *image)
     ImageFormat(image, format);
 }
 
+// Apply box blur
+void ImageBlurBox(Image *image, int blurSize) {
```

Comment:
I'm not familiar with this algorithm so this function is a little hard for me to understand. Is there a description you could add that names it, or even better a link to the algorithm's e.g wikipedia page?

Reply:
https://en.m.wikipedia.org/wiki/Box_blur